core/src/main/scala/ai/h2o/sparkling/H2OContext.scala

```diff
@@ -467,9 +467,14 @@ object H2OContext extends Logging {
     getOrCreate(Option(instantiatedContext.get()).map(_.getConf).getOrElse(new H2OConf()))
   }
 
+  private def isSparkVersionDeprecated(): Boolean = {
+    SparkSessionUtils.active.version.startsWith("2.1")
```

Comment:
What about spark 2.10? ;-)
(I know, it doesn't exist but I just warn about this pattern.)

Reply:
ok, I will improve that